sdc/tests/test_series.py

```diff
@@ -4013,6 +4026,7 @@ def test_impl(series, ascending, kind):
                         np.testing.assert_array_equal(ref_result.data, jit_result.data)
                         self.assertEqual(ref, jit)
 
+    @skip_numba_jit("BUG: sort with kind=mergesort, ascending=False not stable for stirng data (SAT-3828)")
```

Comment:
"not stable for `stirng` data"

Reply:
@AlexanderKalistratov  Yes, thank you. I'm going to unskip it in #961 anyway.